projects/plugins/jetpack/extensions/blocks/paywall/index.js

```diff
@@ -21,26 +21,9 @@ export const settings = {
 	category: 'grow',
```

Comment:
Let's move to `earn` category where other payment related stuff is. Can be in different PR.

Reply:
Moved in https://github.com/Automattic/jetpack/pull/32101/commits/b6c45991b22bd7027a3aa3983153ad4477fda66a